cranelift/codegen/src/isa/x64/inst.isle

```diff
@@ -437,6 +437,19 @@
                  (dst WritableXmm)
                  (src_size OperandSize))
 
+       ;; Conversion from signed integers to floats, the `{v,}`cvtsi2s{s,d}`
+       ;; instructions.
+       (CvtIntToFloat (op SseOpcode)
```

Comment:
I guess the reason we can't use an existing format above is that the sources are Xmm and GprMem rather than, say, Xmm/XmmMem or Gpr/GprMem? Could we have a comment here noting that?